tests/test_warm_reboot.py

```diff
@@ -289,6 +289,9 @@ def stop_neighsyncd(dvs):
 def start_neighsyncd(dvs):
     dvs.runcmd(['sh', '-c', 'supervisorctl start neighsyncd'])
 
+def start_restore_neighbors(dvs):
+    dvs.runcmd(['sh', '-c', 'supervisorctl start restore_neighbors'])
+
```

Comment:
Should a "stop_restore_neighbors(dvs)" function be prepared and used as well?

Reply:
Since restore_neighbors program is one time only and will exit by itself, so I didn't have the stop functions, but for sure we can prepare that function. will add it.